src/core/actions/index.js

```diff
@@ -35,6 +38,15 @@ export function setLang(lang) {
   };
 }
 
+export function setUserAgent(userAgent) {
+  const { browser, os } = UAParser(userAgent);
```

Comment:
```js
    var result = parser.getResult();
    // this will also produce the same result (without instantiation):
    // var result = UAParser(uastring);

    console.log(result.browser);        // {name: "Chromium", version: "15.0.874.106"}
    console.log(result.device);         // {model: undefined, type: undefined, vendor: undefined}
    console.log(result.os);             // {name: "Ubuntu", version: "11.10"}
    console.log(result.os.version);     // "11.10"
    console.log(result.engine.name);    // "WebKit"
    console.log(result.cpu.architecture);   // "amd64"
```

It breaking would be somewhat easy to diagnose as it'd be in a patch that upped the version and we could see where/how we're using `UAParser`. I think it'd be alright, and the syntax I'm using is more concise without giving up anything–the `getBrowser()` code isn't really any better to read.